sechub-doc/src/docs/asciidoc/sechub-quickstart-guide.adoc

```diff
@@ -20,124 +20,89 @@ The following topics are covered:
 
 * [x] Getting SecHub
 * [x] Building SecHub
-* [x] Starting SecHub in Integration Test Server mode
-* [x] Getting default passwords
-* [x] Working with the REST API and https://curl.haxx.se/[cURL]
+* [x] Starting SecHub server in Integration Test mode
+* [x] Default passwords
+* [x] Working with the REST API
+* [x] Creating a project on SecHub server
+* [x] Code scan with the SecHub client
 * [x] Stopping the server
 
 === Requirements
 
-* Java SDK
+* Java 11 SDK
 * Git
 * cURL
+* jq
 
-==== Alpine Linux 3.10, 3.11 & 3.12:
+==== Alpine Linux:
 
-Java 8:
 ----
-apk add openjdk8 curl git bash
+apk add openjdk11 curl git bash jq
 ----
 
-Java 11:
+==== Debian:
```

Comment:
I tested this with Debian 10. I am not sure if Debian 9 or Debian Testing works.

Reply:
My experience is that Debian is mega-consistent across releases.
If somebody downloads Debian, in nearly all cases this will be the current stable version.
I think, we can go with "Debian".